test/GPv2Settlement.test.ts

```diff
@@ -19,10 +19,14 @@ describe("GPv2Settlement", () => {
     settlement = await GPv2Settlement.deploy(uniswapFactory.address);
   });
 
-  describe("DOMAIN_SEPARATOR", () => {
-    it("should have a well defined domain separator", async () => {
-      expect(await settlement.DOMAIN_SEPARATOR()).to.equal(
-        ethers.utils.id("GPv2"),
+  describe("replayProtection", () => {
+    it("should have a well defined replay protection signature mixer", async () => {
+      const { chainId } = await waffle.provider.getNetwork();
```

Comment:
Would you mind asserting that this is nonzero?

Reply:
Good idea!

Reply:
Might be nice to test that multiple deployed instances have different replay protections.